environments/qa/modules/oaeqaautomation/templates/redeploy.sh.erb

```diff
@@ -29,43 +42,62 @@ fi
 echo "Shutting down services..."
 <%= scripts_dir %>/shutdown.sh
 
-echo ""
+echo
 echo "Deleting data..."
 <%= scripts_dir %>/deletedata.sh
 
-echo ""
+# I'm having less hassles when the app & ux dirs are removed
+rm -rf "<%= app_root_dir %>" "<%= ux_root_dir %>"
```

Comment:
This can be removed as the `deletedata.sh` script that runs just before this takes care of this rimraffing:

https://github.com/oaeproject/puppet-hilary/blob/master/environments/qa/modules/oaeqaautomation/templates/deletedata.sh.erb#L23-L25